linkerd/client-policy/src/lib.rs

```diff
@@ -0,0 +1,146 @@
+#![deny(rust_2018_idioms, clippy::disallowed_methods, clippy::disallowed_types)]
+#![forbid(unsafe_code)]
+
+pub mod http;
+
+#[cfg(feature = "proto")]
+pub mod proto;
+
+use linkerd_addr::{Addr, NameAddr};
+// pub use linkerd_policy_core::{meta, Meta};
+use linkerd_proxy_api_resolve as resolve;
+use std::{fmt, net::SocketAddr, str::FromStr, sync::Arc};
+use tokio::sync::watch;
+
+type Backends = Arc<[Backend]>;
+
+#[derive(Clone, Debug)]
+pub struct Receiver {
+    inner: watch::Receiver<Policy>,
+}
```

Comment:
let's do without this stuff for now and focus on just the `Policy` type for the time being. (I think the profiles::Receiver type isn't so useful in general)

Reply:
yeah, i thought about getting rid of both of them